### DISCUSSION TITLE
Showing F# errors in the Vite error overlay.

### DISCUSSION THREAD

**Discussion by colelawrence** [nojaf/vite-plugin-fable#7] [Ideas]
It would be special to see F# syntax errors show up in the Vite error overlay.
![2024-05-16 index js — phosphornext cldev (Workspace) at 05 11PM@2x](https://github.com/nojaf/vite-plugin-fable/assets/2925395/cfc86b6e-67c8-4cfa-92d1-7b8a27514f65)

As a side note, when there is a syntax error, something like HMR still seems to try refreshing which spins into a loop of unhandled promise errors.

**Comment by nojaf** [comment]
Hi,

Interesting suggestion. I quickly played around with this:

![image](https://github.com/nojaf/vite-plugin-fable/assets/2621499/e1b7c319-fadb-4f0b-bda9-5dedfa9388eb)

It is possible. Need to play with this some more.

**Comment by nojaf** [comment]
Initial support for this is available in https://www.npmjs.com/package/vite-plugin-fable/v/0.0.29

**Reply by colelawrence** [reply]
Thanks, I'll test it out today!
